packages/extension-ui/src/Popup/index.tsx

```diff
@@ -51,7 +51,7 @@ async function requestMediaAccess (cameraOn: boolean): Promise<boolean> {
 
 function initAccountContext (accounts: AccountJson[]): AccountsContext {
   const hierarchy = buildHierarchy(accounts);
-  const master = hierarchy.find((account) => !account.isExternal);
+  const master = hierarchy.find((account) => !account.isExternal && account.type !== 'ethereum');
```

Comment:
Thinking about it, I'll rather go for `['ed25519', 'sr25519', 'ecdsa'].includes(type)` to be future proof like I did elsewhere